app/bergamot-translator-app-bytearray.cpp

```diff
@@ -20,19 +20,20 @@ int main(int argc, char **argv) {
   std::string config = options->asYamlString();
 
   // Route the config string to construct marian model through TranslationModel
-  TranslationModel model(config, marian::bergamot::getModelMemoryFromConfig(options));
+  marian::bergamot::Service model(
+      config, marian::bergamot::getModelMemoryFromConfig(options));
```

Comment:
We should also pass shortlist by bytes while calling this constructor. Right? Otherwise, it will be using only model as bytes but shortlist will still be loaded from file.

Reply:
I'd prefer to have @qianqianzhu bring this change in a separate PR (she's working on vocabs and will tread the bytearray test-app territory meanwhile). Does this suggestion work for everyone? 

I'm not changing anything here except my 80 column wrap setting running over Nick's settings, if a closer look is taken.

Reply:
Yes. Better leave it to me as I am adding vocab bytes as a new argument here.